src/templates/users/user_list.html

```diff
@@ -0,0 +1,15 @@
+{% extends "base.html" %}
```

Comment:
Same comment as the `user_list` view goes here. Why is this template here when DRF autogenerates a reasonable view with the user of `rest_framework.generics.ListAPIView`.

Reply:
That file has been removed